social-team.md

```diff
@@ -0,0 +1,72 @@
+
+The Social Team is tasked with maintaining the project social accounts. This includes:
+
+- content creation
+- amplification of our community
+  - sharing, retweeting, and so on
+- engagement with our community
+  - liking, replying, and so on
+- updating account metadata (like bio, avatar, header images, and so on)
+
+With regard to posts and community interaction, there is not requirement or entitlement - rather, posts and interactions are completely at the discretion of the Social Team's members.
+
+### Platforms 
+
+This team controls and maintains the various social platforms that Node.js has:
+
+- [Node.js on Twitter](https://twitter.com/nodejs)
+- [Node.js on Facebook](https://www.facebook.com/nodejsfoundation)
+- [Node.js on LinkedIn](https://www.linkedin.com/company/node.js-foundation/)
+
+### Access
+
+Social Team members will have full access to all [platforms](#platforms).
+
+### Expectations
+
+The basic expectations are that **all** Social Team members will have both a willingness and ability to...
+
+- spend time weekly engaging on various platforms, or communicate the inability to do so with other social team members and work with them to ensure there is *some* coverage
+- spend time working with other social team members on building out good content
+- spend time working work project committees, working groups, initiatives, and teams to build effective outward communication as needed
+
+### Repositories
+
+The Social Team will be responsible for and maintain the following:
+
+- [nodejs/tweet](http://github.com/nodejs/tweet): a repo leveraging [Twitter Together](https://github.com/gr2m/twitter-together) that enables project members to queue up tweets to [@nodejs](https://twitter.com/nodejs) from within GitHub.
+
+## Membership
+
+### Nominations
+
+Nominations and self-nominations are open to **project members** on a rolling basis. An issue should be created in [nodejs/admin](https://github.com/nodejs/admin) for any kind of nomination.
+
+### Objections
+
+Membership in the Social Team **requires** zero objections from [TSC](https://github.com/nodejs/tsc) and [CommComm](https://github.com/nodejs/community-committee) members.
+
+Should there be an objection from a TSC member or CommComm member to a nomination, there are multiple paths to surfacing it:
+
+- Directly in a nomination issue
+- Through requesting a peer on the TSC or CommComm proxy your objection
+- Through the TSC or CommComm chairperson who will proxy your objection
+
+An objection *does* require substantiation beyond a simple `-1`, which can be commented or otherwise passed along through an objection proxy. Since this is a role built upon extreme trust and high impact, the substantiation is not negotiable by either the nominee or other TSC or CommComm members.
+
+More explicitly: If an objection is stated, the nomination does not pass.
+
+#### Re-nomination
+
+If a nomination is objected to, the nominee can be re-nominated or re-self-nominated after 6 months.
+
```

Comment:
Do we need a section on "removal" ?  Hopefully it never occurs, but probably good to cover anyway. Possibly:

### Removal

A member of the Social team may be removed on request of any TSC or CommComm member.  This request can be raised in the same way as listed for raising an objection to a nomination.

Reply:
Added, with an additional note that it should be done in the original nomination issue.